code/modules/food_and_drinks/machinery/microwave.dm

```diff
@@ -575,7 +575,7 @@
 
 	if(cell_powered && cell?.charge < TIER_1_CELL_CHARGE_RATE * efficiency)
 		playsound(src, 'sound/machines/buzz-sigh.ogg', 50, FALSE)
-		balloon_alert(cooker, "replace cell!")
+		balloon_alert(cooker, "no cell!")
```

Comment:
I think the wording on this can be tweaked, because it will show in both cases the cell is empty, or the cell is missing.